library/kotlin/src/io/envoyproxy/envoymobile/filters/FilterDataStatus.kt

```diff
@@ -14,7 +14,7 @@ sealed class FilterDataStatus {
    * before the entirety is sent to the next filter.
    * TODO: add param docs. name param in swift.
```

Comment:
Might be worth resolving this while we're here, can also do separately

Reply:
👍